core/src/main/java/entities/HeatPoint.java

```diff
@@ -0,0 +1,55 @@
+package entities;
+
+import java.util.Map;
+
+public class HeatPoint {
+
+    private static double radius = 0.0013;
+
+    private double x;
+
+    private double y;
+
+    private Map<String, Object> attributes;
+
+    public HeatPoint(){}
+
+    public HeatPoint(double x, double y) {
+        this.x = x;
+        this.y = y;
+    }
+
+    public double getX() {
+        return x;
+    }
+
+    public void setX(double x) {
+        this.x = x;
+    }
+
+    public double getY() {
+        return y;
+    }
+
+    public void setY(double y) {
+        this.y = y;
+    }
+
+    public Map<String, Object> getAttributes() {
+        return attributes;
+    }
+
+    public void setAttributes(Map<String, Object> attributes) {
```

Comment:
this can be replaced by method which just add new attribute, so no need to create map and set it again